models/DataObject/Data/QuantityValueRange.php

```diff
@@ -64,8 +64,19 @@ public function setMaximum(int|float|null $maximum): void
         $this->markMeDirty();
     }
 
-    public function getRange(int $step = 1): array
+    public function getRange(int|float $step = 1): array
```

Comment:
Thank you for your PR!

This change would be a `bc-break`, any extending class would not be compatible to this signature.
Could you please turn this PR into an Improvement for `12.x` and eventually add a changelog entry about the support for negative steps? TIA

Reply:
feel free to open a new PR and close this one